tests/financial/test_national_stock_exchange_of_india.py

```diff
@@ -218,10 +238,51 @@ def test_guru_nanak_jayanti(self):
             "2017-11-04",
         )
 
+    def test_bhai_bij(self):
+        name = "Bhau Bhij"
+        self.assertNonObservedHolidayName(
+            name,
+            "2003-10-26",
+            "2005-11-03",
+            "2006-10-24",
+            "2008-10-30",
+            "2009-10-19",
+        )
+        self.assertNonObservedHolidayName(name, range(2003, 2010))
+        self.assertNoHoliday(
+            "2010-10-06",
+            "2002-11-05",
+        )
+
+    def test_buddha_purnima(self):
+        name = "Buddha Purnima"
+        self.assertNonObservedHolidayName(
+            name,
+            "2007-05-02",
+            "2008-05-19",
+        )
+        self.assertNonObservedHolidayName(name, range(2006, 2010))
+        self.assertNoHoliday(
+            "2025-05-19",
+        )
+
     def test_muharram(self):
         name = "Muharram"
         self.assertNonObservedHolidayName(
             name,
+            "2001-04-05",
+            "2002-03-25",
+            "2004-03-02",
+            "2007-01-30",
+            "2009-01-08",
+            "2009-12-28",
+            "2010-12-17",
+            "2011-12-06",
+            "2013-11-14",
+            "2014-11-04",
+            "2016-10-12",
+            "2019-09-10",
+            "2021-08-19",
```

Comment:
```suggestion
            "2020-08-30",
```
2020-2025 is standard range for Islamic holidays tests.